From 2f5514051210388bfcff605570d33f08cfa7bcaa Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Wed, 21 Jul 2021 08:34:58 +0000
Subject: [PATCH] IWYU: usage of unique_ptr requires including <memory> in
 bluetooth low energy scan filter.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix build because of missing include:
../../device/bluetooth/bluetooth_low_energy_scan_filter.h:57:15: error: ‘unique_ptr’ in namespace ‘std’ does not name a template type
   57 |   static std::unique_ptr<BluetoothLowEnergyScanFilter> Create(
      |               ^~~~~~~~~~

Bug: 819294
Change-Id: I347953a083f1bcdf744fd86e1a73954c6f86b32e
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3041155
Reviewed-by: Reilly Grant <reillyg@chromium.org>
Commit-Queue: José Dapena Paz <jdapena@igalia.com>
Cr-Commit-Position: refs/heads/master@{#903819}
---
 device/bluetooth/bluetooth_low_energy_scan_filter.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/device/bluetooth/bluetooth_low_energy_scan_filter.h b/device/bluetooth/bluetooth_low_energy_scan_filter.h
index a0436c184a967..7ae606cca014e 100644
--- a/device/bluetooth/bluetooth_low_energy_scan_filter.h
+++ b/device/bluetooth/bluetooth_low_energy_scan_filter.h
@@ -7,6 +7,7 @@
 
 #include <stddef.h>
 #include <stdint.h>
+#include <memory>
 #include <vector>
 
 #include "base/time/time.h"
